database/mysql/0013_user_management.sql

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Copyright (C) 2017-2018 Eaton
+ *
+ * This software is confidential and licensed under Eaton Proprietary License
+ * (EPL or EULA).
+ *
+ * This software is not authorized to be used, duplicated or disclosed to
+ * anyone without the prior written permission of Eaton.
+ * Limitations, restrictions and exclusions of the Eaton applicable standard
+ * terms and conditions, such as its EPL and EULA, apply.
+ *
+ */
+
+/*
+ * @file    0013_user_management.sql
+ * @author: <a href="mailto:XavierMillieret@Eaton.com">Xavier, Millieret (E9902068)</a>
+ * @brief   User management data base definition. 
+ */
+
+
+/* For details on schema version support see the main initdb.sql */
+SET @bios_db_schema_version = '201802140001';
+SET @bios_db_schema_filename = '0013_user_management.sql';
+
+use box_utf8;
+
+/* This should be the first action in the SQL file */
+START TRANSACTION;
+INSERT INTO t_bios_schema_version (tag, timestamp, filename, version) VALUES('begin-import', UTC_TIMESTAMP() + 0, @bios_db_schema_filename, @bios_db_schema_version);
+/* Report the value */
+SELECT * FROM t_bios_schema_version WHERE tag = 'begin-import' order by id desc limit 1;
+COMMIT;
+
+/*
+*
+* User management tables creations 
+*
+*
+*/
+
+/* User table */
+CREATE TABLE IF NOT EXISTS t_user(
```

Comment:
I believe we already have a sort of table for tracking user accounts of the REST API/web-based GUI, with current stress on personal profile settings: `t_bios_agent_info`

Is there some strong case for a separate table, and for binding with the particular deployment's Unix UIDs at all? Note that eventually we might add e.g. LDAP/AD/... logins for the REST API, which should not necessarily be Unix logins at all (though they can, as implementation detail or otherwise). Or perhaps some central node would tell RC instances to provision accounts - and that might be based on the first available UID number on a particular system, so nearly-inevitably the user:number mappings would differ across the farm over time).

Reply:
For me, and after discussion with Gerald, this table was here to store agents configuration, the desc of this table is:
+------------+----------------------+------+-----+---------+----------------+
| Field      | Type                 | Null | Key | Default | Extra          |
+------------+----------------------+------+-----+---------+----------------+
| id         | smallint(5) unsigned | NO   | PRI | NULL    | auto_increment |
| agent_name | varchar(50)          | NO   | UNI | NULL    |                |
| info       | blob                 | NO   |     | NULL    |                |
+------------+----------------------+------+-----+---------+----------------+
And for my case I think, this table is not adapt (semantic) for my case